.env.example

```diff
@@ -0,0 +1,4 @@
+DB_HOST =
+DB_USER =
+DB_PASS =
+DB_DATABASE =
```

Comment:
Why are you sending .env file on github.

Reply:
It is an .env.example file to let anyone know what environment variables are used.